CMakePresets.json

```diff
@@ -127,6 +127,15 @@
                 "CMAKE_CXX_COMPILER": "/usr/bin/g++"
             }
         },
+        {
+            "name": "windows-mingw-release",
```

Comment:
not sure why you added it if it doesn't actually run on CI. Also it should use Conan to use pre-built binaries.

Reply:
Using CMake presets have some advantages over old-school CMake with flags. Most important advantage is IDE support in VSCode and Visual Studio. So it has value even if CI doesn't run preset.

Not all build configurations use conan, right? Dependencies can easily be installed via MSYS2's `pacman`

Reply:
then this deserves a documentation how to use MinGW without Conan (i.e. how to obtain dependencies): point to msys2, list which packages must be installed etc.

with the current PR we'll have a preset that nobody knows how to use

Reply:
good point, I'll make a new commit that updates documentation on how to build with this preset

Reply:
> good point, I'll make a new commit that updates documentation on how to build with this preset
> 
> **EDIT:** Added with [a78a3b3](https://github.com/vcmi/vcmi/pull/3421/commits/a78a3b3e4fb505966c07de50e4b2598b32319a40)

thanks!